Client/game_sa/CPedSA.h

```diff
@@ -258,7 +263,8 @@ class CPedSAInterface : public CPhysicalSAInterface            // +1420  = curre
     // weapons at +1440 ends at +1804
     BYTE                bPad4[12];
     BYTE                bCurrentWeaponSlot;            // is actually here
-    BYTE                bPad6[20];
+    BYTE                bPad6[3];
```

Comment:
May be `BYTE bPad6[16];` @FileEX

Reply:
No, I just forgot to add padding after pTargetedObject